BuildLoader.py

```diff
@@ -190,6 +190,7 @@ def __init__(self, *args, **kwargs):
 		self.ENABLE_GRUB_CONFIG    = 0
 		self.ENABLE_SMBIOS         = 0
 		self.ENABLE_LINUX_PAYLOAD  = 0
+		self.ENABLE_CONTAINER_BOOT = 1
```

Comment:
Enabled by default?

Reply:
Moving forward, the plan is to retire IAS, so making container image boot as default.

Reply:
Got it. Thanks.